src/main/studio/kdb/Config.java

```diff
@@ -178,14 +179,34 @@ private String getWorkspaceFilename() {
 
     public Workspace loadWorkspace() {
         Workspace workspace = new Workspace();
-        File workspaceFile = new File(getWorkspaceFilename());
+        String fileName = getWorkspaceFilename();
+        File workspaceFile = new File(fileName);
+        boolean loadBackup = false;
         if (workspaceFile.exists()) {
-            try (InputStream inp = new FileInputStream(workspaceFile)) {
+            try (InputStream inp = Files.newInputStream(workspaceFile.toPath())) {
                 Properties p = new Properties();
                 p.load(inp);
                 workspace.load(p);
+                if (!p.containsKey("zzzzz")) {
```

Comment:
I propose to have some meaningful name (like saveControlFlag ? ); and extract to constant (private static final variable) to reference to it in another method.